scripts/nat/start-nat-node.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+if [ $(id -u) -ne 0 ] ; then
+  >&2 echo "ERROR: Must run as root"
+  exit 1
+fi
+
+if [ -z "$HOPR_RELEASE" ]; then
+  >&2 echo "ERROR: Specify HOPR_RELEASE environment variable"
+  exit 1
+fi
+
+if [ ! -S "/var/run/docker.sock" ]; then
+  >&2 echo "ERROR: /var/run/docker.sock must be mounted"
+  exit 1
+fi
+
+readonly network_name="hopr-nat"
+
+if [ "$(docker network ls | grep -c "$network_name" )" = "0" ] ; then
+  if ! docker network create -d bridge hopr-nat > /dev/null 2>&1 ; then
+    >&2 echo "ERROR: Failed to create network for NAT"
+    exit 1
+  fi
+fi
+
+# Fork here and pass all the environment variables down into the forked image
+env > env.list
```

Comment:
security risk, at the very least the files should be deleted afterwards

Reply:
Yes, I forgot about that one. Original plan was to grep for only a few relevant env vars (non-sensitive).

Reply:
Replaced with process substitution bash-ism.